lcdb_test_data/Snakefile

```diff
@@ -57,32 +72,102 @@ rule flux:
                 READ_NUMBER	5000
                 ERR_FILE	76
                 FASTA	YES
+                PAIRED_END	true
+                UNIQUE_IDS	true
                 SEED	{sample_int}
                 """.format(**locals())))
         shell(
             'flux-simulator --force -t simulator -x -l -s -p {fout.name} &> {log}')
 
+        # flux-simulator creates an interleaved fastq. So we deinterleave, idea
+        # from https://gist.github.com/nathanhaigh/3521724
+        #
+        # Note that if UNIQUE_IDS is false (by default), read names end with
+        # "S/2" or "A/1" and "S" or "A" indicate "sense" or "antisense". When
+        # UNIQUE_IDS is true (as we use here), they only have /1 or /2, and
+        # read 1 is always sense while read 2 is antisense.
+        #
+        # Read more by checking `flux-simulator --printParameters`.
+        #
+        fastq = 'samples/{sample}/{sample}.fastq'.format(sample=wildcards.sample)
+        shell(
+            'cat {fastq} '
+            '| paste - - - - - - - - '
+            '| tee '
+            '>(cut -f 1-4 | tr "\t" "\n" | gzip -c > {output.fastq_R1}) '
+            '| cut -f 5-8 | tr "\t" "\n" | gzip -c > {output.fastq_R2}'
+        )
```

Comment:
(the new things for me were `paste` joining 8 lines at a time by tabs, and using `tee` with a process substitution. `tee` sends input to file and to stdout. The first chunk that would normally go to file is instead sent to a process substitution that cuts the first cols; the second chunk goes to stdout and so can be simply piped.)